app-office/taskcoach/taskcoach-1.4.6.ebuild

```diff
@@ -0,0 +1,45 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+PYTHON_COMPAT=( python2_7 )
+
+inherit desktop distutils-r1
+
+MY_PN="TaskCoach"
+MY_P="${MY_PN}-${PV}"
+
+DESCRIPTION="Simple personal tasks and todo lists manager"
+HOMEPAGE="https://www.taskcoach.org https://pypi.org/project/TaskCoach/"
+SRC_URI="mirror://sourceforge/${PN}/${MY_P}.tar.gz"
+LICENSE="GPL-3+"
+
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE=""
+
+# Task Coach can also optionally use igraph to draw task dependency graphs
+# since 1.4.4, but that is not packaged in Gentoo at this time.
+
+DEPEND="
+	dev-python/wxpython:3.0[${PYTHON_USEDEP}]
+	|| (
+		>=dev-python/twisted-16.0.0[${PYTHON_USEDEP}]
+		>=dev-python/twisted-core-10.0
```

Comment:
I feel like you should still add python_usedep here, in case python3 support is added at some point.

Reply:
Yes, I agree; I just missed adding it when copying from 1.4.3-r1. Done in fixup commit 030419e70861.